src/main/asciidoc/micrometer/prometheus/main.adoc

```diff
@@ -0,0 +1,171 @@
+| tag name | SCDF property | default value
+| stream.name |  spring.cloud.dataflow.stream.name | unknown
+| application.name | spring.cloud.dataflow.stream.app.label | unknown  
+| instance.index | instance.index | 0 
+| application.guid | spring.cloud.application.guid | unknown 
+| application.gype | spring.cloud.dataflow.stream.app.type | unknown
+|====================
+
+NOTE: For custom app starters that don't extend from the https://github.com/spring-cloud-stream-app-starters/core[core] parent, you should add the `app-starters-common` : `org.springframework.cloud.stream.app` dependecny to enable the SCDF tags.
+
+Prometheus employs the pull-metrics model, called metrics scraping. Spring Boot provides an actuator endpoint available at `/actuator/prometheus` to present a Prometheus scrape with the appropriate format.
+
+Furthermore Prometheus requires a mechanism to discover the target applications to be monitored (e.g. the URLs of the SCSt app intances). Targets may be statically configured via the `static_configs` parameter or dynamically discovered using one of the supported service-discovery mechanisms.
+
+The https://github.com/tzolov/spring-cloud-dataflow-prometheus-service-discovery[SCDF Prometheus Service Discovery] is a standalone (Spring Boot) service, that uses the https://goo.gl/kE4eLV[runtime/apps] endpoin to retrieve the URLs of the running SCDF applications and generate `targets.json` file. The targets.json file is compliant with the  https://prometheus.io/docs/prometheus/latest/configuration/configuration/#%3Cfile_sd_config%3E[<file_sd_config>] Prometheus discovery fromat.
+ 
+Below we will present the steps to prep, configure the demo of Spring Cloud Data Flow's `Local` server integration with `Prometheus`. For other deployment environment, such as `Cloud Foundry` or `Kubernetes`, addition configurations might be required.
+
+==== Prerequisites
+
+* A Running Data Flow Shell
+include::{docs_dir}/shell.adoc[]
+* A running local Data Flow Server
+include::{docs_dir}/local-server.adoc[]
+* Running instance of link:http://kafka.apache.org/downloads.html[Kafka]
+* Spring Cloud Stream 2.x based https://github.com/spring-cloud-stream-app-starters/time/blob/master/spring-cloud-starter-stream-source-time/README.adoc[Time] and  https://github.com/spring-cloud-stream-app-starters/log/blob/master/spring-cloud-starter-stream-sink-log/README.adoc[Log] appcliations starters, pre-build with `io.micrometer:micrometer-registry-prometheus` dependency.
++
+NOTE: Next versions of the https://start-scs.cfapps.io/[SCSt App Initializr] utility would add support for Micrometer dependecies to facilitate the injection of micrometer-registries with SCSt apps. 
+
+==== Building and Running the Demo
+
+. Register `time` and `log` applications pre-build `io.micrometer:micrometer-registry-prometheus`. Coming with the next version of https://start-scs.cfapps.io/[SCSt App Initializr] is support for adding Micrometer registry dependencies as well.
```

Comment:
These statements can be modified similar to the InfluxDB/Grafana page.

Reply:
+